e2xgrader/preprocessors/saveautogrades.py

```diff
@@ -5,8 +5,19 @@
 
 from ..utils.extra_cells import determine_grade
 
+from traitlets import Unicode, List
+from textwrap import dedent
 
 class SaveAutoGrades(NbgraderSaveAutoGrades):
+ 
+    cell_id = Unicode(
```

Comment:
I would use a list here and call this cell_ids

```
cell_ids = List(
    help=...
).tag(config=True)
```

Reply:
Replaced Unicode with List.